src/systems/filecoin_vm/runtime/impl/runtime.go

```diff
@@ -363,7 +363,7 @@ func (rt *VMContext) _transferFunds(from addr.Address, to addr.Address, amount a
 	return nil
 }
 
-func (rt *VMContext) _getActorCodeID(actorAddr addr.Address) (ret actor.CodeID, ok bool) {
+func (rt *VMContext) GetActorCodeID(actorAddr addr.Address) (ret actor.CodeID, ok bool) {
```

Comment:
Objection! Actor have no good reason to inspect the code type of another actor.

Reply:
(Addressed via out-of-band discussions.)